aws-storage-s3/src/main/res/drawable/ic_notification_test.xml

```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
We should rename to "amplify_storage_transfer_notification_icon.xml" so that we are properly namespaced to Amplify. This prevents collision with other libraries, and allows the customer a clear path to override this drawable with a different image.